cirq/testing/random_circuit_test.py

```diff
@@ -72,24 +72,31 @@ def test_random_circuit(n_qubits: Union[int, Sequence[cirq.Qid]],
 
 @pytest.mark.parametrize('seed', [randint(0, 2**32) for _ in range(10)])
 def test_random_circuit_reproducible_with_seed(seed):
-    circuit1 = random_circuit(qubits=20,
-                              n_moments=20,
-                              op_density=0.7,
-                              random_state=seed)
-    circuit2 = random_circuit(qubits=20,
-                              n_moments=20,
-                              op_density=0.7,
-                              random_state=seed)
-    assert circuit1 == circuit2
+    seeds = (seed, np.random.RandomState(seed))
+    circuits = [
+        random_circuit(qubits=20,
+                       n_moments=20,
+                       op_density=0.7,
+                       random_state=seed) for seed in seeds for _ in range(2)
+    ]
+    eq = cirq.testing.EqualsTester()
+    eq.add_equality_group(circuits)
```

Comment:
This call is incorrect; it should be
```
eq.add_equality_group(*circuits)
```
Fixing this reveals that the test is incorrect because it passes the same RandomState object twice in a row, which results in different circuits. That's why in the original test, the line
```
prng = np.random.RandomState(seed)
```
is performed twice.

Reply:
Fixed. Two bugs makes a right :)